src/main/java/uk/gov/hmcts/reform/civil/HighLevelDataSetupApp.java

```diff
@@ -103,61 +80,4 @@ public void createRoleAssignments() {
         // Do not create role assignments.
         BeftaUtils.defaultLog("Will NOT create role assignments!");
     }
-
-    @Override
-    protected boolean shouldTolerateDataSetupFailure(Throwable e) {
-        int httpStatusCode504 = 504;
-        if (e instanceof ImportException) {
-            ImportException importException = (ImportException) e;
-            return importException.getHttpStatusCode() == httpStatusCode504;
-        }
-        return false;
-    }
```

Comment:
Mindful with removing this one: its purpose was to allow the import to continue when an HTTP-504 error happened, as that error can be tolerated.

Not sure if the removal was intentional or not, so just highlighting this.

Reply:
@filippo-possenti-version1 I'll put this back since it seems import exception is part of the befta library, sorry for the confusion!